sway/input/cursor.c

```diff
@@ -139,6 +161,11 @@ struct sway_node *node_at_coords(
 		return NULL;
 	}
 
+	if ((*surface = input_popup_surface_at(output,
```

Comment:
We should probably move this up right before "check for unmanaged views" to allow fullscreen surfaces to work with IMEs.